src/settings/MappingProfiles/MappingProfilesTransformationsModal/MappingProfilesTransformationsModal.test.js

```diff
@@ -171,33 +171,46 @@ describe('MappingProfilesTransformationsModal', () => {
           expect(transformationFields[1].isInvalid).toBe(false);
         });
 
-        it('should validate item record type transformation as invalid', () => {
+        it('should validate item record type transformation and mark marcField as invalid', () => {
           expect(transformationFields[2].isInvalid).toBe(true);
+          expect(transformationFields[2].marcField.isInvalid).toBe(true);
+          expect(transformationFields[2].indicator1.isInvalid).toBe(false);
+          expect(transformationFields[2].indicator2.isInvalid).toBe(false);
+          expect(transformationFields[2].subfield.isInvalid).toBe(false);
         });
 
         describe('filling empty invalid field with invalid transformation', () => {
           beforeEach(() => {
```

Comment:
As I see there are a couple of cases when there is only one test after `beforeEach` which means the describe block can be omitted at all. Overall it looks good from the readability perspective but as I remember when we have such cases we just eliminate such describes.

Reply:
I think we should leave it like that for readability. I don't think it does any harm to anything, but if we would merge `beforeEach` and `it` block it would look quite chunky. What do you think?

Reply:
Sounds good to me. Let's agree when `beforeEach` and `it` blocks are quite big we proceed with the current approach.